service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskExecutor.java

```diff
@@ -26,6 +29,10 @@
     private GpcClient gpcClient;
     @Autowired
     private DetectTranslationCompleteService detectTranslationCompleteService;
+    @Autowired
+    private MhsPayloadMapper mhsPayloadMapper;
```

Comment:
What do you think about changing this class this time to use lombok @AllArgsConstructor(onConstructor = @__(@Autowired)) and make all dependencies final ?